### DISCUSSION TITLE
Camera2 not following when playing moving in a NE map.

### DISCUSSION THREAD

**Discussion by Stormanzanii** [kinsi55/CS_BeatSaber_Camera2#107] [Q&A] [answered]
When i play a NE map that requires movement, the camera just stays at were it was once it started and never attached to the player, only thing that was attached was the rotations.

**Comment by kinsi55** [comment, marked as answer]
Update to the latest version in this Repo (Its not in Modassistant yet)

**Reply by Stormanzanii** [reply]
what if im not using mod assistant? as im currently using BSManager or some deal?

**Reply by kinsi55** [reply]
There is only one repository for mods, Beatmods - All the mod managers pull from that

**Reply by Stormanzanii** [reply]
so.. is that a yes?

**Reply by kinsi55** [reply]
A yes to what? For now you need to manually grab the update from the releases here.

**Reply by Stormanzanii** [reply]
oh oki

**Reply by Stormanzanii** [reply]
I TURNED OFF THE UI AND NOW I CAN'T GET IT BACK-

**Reply by kinsi55** [reply]
Do it ingame
